lib/spack/spack/ci/common.py

```diff
@@ -33,14 +34,34 @@
 from spack.reporters.cdash import SPACK_CDASH_TIMEOUT
 from spack.reporters.cdash import build_stamp as cdash_build_stamp
 
+try:
+    import gzip  # noqa
```

Comment:
we import this in various places unconditionally

Reply:
I copied it from `spack.util.compression` but if it is assumed true then I will drop it.

Reply:
see installer.py for example

Reply:
I am skipping the extra bits to omit the header info since I don't think it really matters for this stuff.